### ISSUE TITLE
Check IIIF-URLs behaviour

### ISSUE THREAD

**Issue by riedde** [Edirom/Edirom-Online#231]
In a discussion about mei metadata and image urls I got a hint about the use of [IIIF- URLs](https://github.com/music-encoding/metadata-ig/issues/8#issuecomment-1128725620)

This should be reflected and documented!

**Comment by bwbohl** [comment]
I don't think that we would have to document this in the context of Edirom-Online, but at least make sure we expect the right thing and maybe point out the MEI documentation. Does that sound plausible?

**Comment by riedde** [comment]
Yes. For me it was new and I noticed that I sometimes use this in a false way. If both works it is okay for me I thought that we just document it if a special way of use is required because of the openseadragon implementation in the edirom

**Comment by bwbohl** [comment]
We should definitely assure the media fragment syntax works an if needed translate it to the Image API syntax.
